src/components/site-footer/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { FOOTER_NAV_LINKS } from "@/config/navigation";
+import Link from "next/link";
+import { FaMoneyCheckDollar } from "react-icons/fa6";
+import { Separator } from "../ui/separator";
+import { BsTwitterX } from "react-icons/bs";
+import { SlSocialFacebook, SlSocialInstagram } from "react-icons/sl";
+
+const SiteFooter = () => {
+  return (
+    <footer className="container mt-[50px] border-t py-[50px]">
+      <div className="flex flex-col md:flex-row">
+        <div className="mb-[20px] md:mr-[40px] md:grow-[1]">
+          <Link href="/" className="flex flex-col max-w-max">
+            <FaMoneyCheckDollar className="mr-[12px] text-[60px] md:text-[40px]" />
+            <span className="font-[600] text-[40px] md:text-[30px]">SpendWise</span>
+            <p className="text-foreground/40">Finance at your control.</p>
+          </Link>
+        </div>
+        <div className="grid gap-[20px] xxs:grid-cols-2 sm:grid-cols-3 md:grow-[3]">
+          {FOOTER_NAV_LINKS.map((nav) => (
```

Comment:
by default your device will be in mobile so no need xxs:grid-cols-2 directly you can write grid-cols-2. what will be after xxs device. tailwind is mobilefirst so what ever you write it will directly apply to mobile that's the reason you don't need xxs: .

Reply:
@DhanBirSunwar since by default `display:grid` results in single column which makes sense for smaller screens, but after `360px`, need two columns hence `xxs:grid-cols-2`